manifests/manage/resource.pp

```diff
@@ -1,21 +1,25 @@
-# == Define: hbm::manage::resource
+# @param ensure
+#   TODO: Add documentation
```

Comment:
TODO? :)

Reply:
There need to be a description otherwise puppet-lint will fail.
I am not able to provide documentation myself for these cases.

Reply:
Can you reach out to Daniel T? He should be able to help you out with expanding the description of these parameters.

Reply:
I have sent a mail to Daniel and hope for the best...

Reply:
Daniel reached back to me and provided the missing documentation and I have added it accordingly.